src/Sarif/VersionOne/Readers/SarifContractResolverVersionOne.cs

```diff
@@ -53,9 +53,6 @@ protected override JsonContract CreateContract(Type objectType)
             else if (objectType == typeof(SuppressionStatesVersionOne))
                 contract.Converter = FlagsEnumConverter.Instance;
 
-            else if (objectType == typeof(Dictionary<string, IRuleVersionOne>))
```

Comment:
>IRuleVersionOne [](start = 61, length = 15)

This can't be right -- we didn't (or shouldn't have) removed `IRule` from version 1.

Reply:
this is right! this converter existed due to a bug in JSON.NET v6 which we no longer have a dependency on. So this converter can disappear entirely.

---
In reply to: [255308073](https://github.com/Microsoft/sarif-sdk/pull/1259#discussion_r255308073) [](ancestors = 255308073)